app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -301,7 +302,8 @@ class SettingsFragment : PreferenceFragmentCompat() {
     private fun setupPreferences() {
         val leakKey = getPreferenceKey(R.string.pref_key_leakcanary)
         val debuggingKey = getPreferenceKey(R.string.pref_key_remote_debugging)
-
+        val externalDownloadManagerKey = getPreferenceKey(R.string.pref_key_external_download_manager)
+        val preferenceExternalDownloadManager = findPreference<Preference>(externalDownloadManagerKey)
```

Comment:
I added a `requirePreference` function you can use instead, since this should never be null.

Reply:
👍